### PR TITLE
layout: created BaseInput

### PR DESCRIPTION
**Pull request by vlad99902**
Created new component. Team lead lord, please merge it.

### CODE REVIEW

File: src/components/BaseInput.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { InputHTMLAttributes } from "react";
```
Comment (bacebu4) [inline src/components/BaseInput.tsx:1]: Unnecessary `React` import

File: src/components/BaseInput.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import styled from "styled-components";
+import { GRAY, DARK } from "../utils/colors";
+
+const InputArea = styled.input<{ margin?: string }>`
+  margin: ${(props) => props.margin};
```
Comment (bacebu4) [inline src/components/BaseInput.tsx:6]: In this project we're using `mt` and `ml`
For me it's easier to read
Even though this solution is more flexible let's stick to `mt` and `ml`

File: src/components/BaseInput.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  font-size: 1rem;
+  padding: 8px 12px;
+  background-color: ${GRAY}1A;
+  border-radius: 9px;
+  border: 1px solid ${GRAY}03;
```
Comment (bacebu4) [inline src/components/BaseInput.tsx:12]: `1A` and `03` looks like a _magic_ to me
Probably don't need to hard code them
I'll add the issue about that, don't need to fix it for you

File: src/components/BaseInput.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  outline: none;
```
Comment (bacebu4) [inline src/components/BaseInput.tsx:13]: Don't use it. We're making our site accessible :)

**Review by bacebu4 — CHANGES_REQUESTED** [review]

File: src/components/BaseInput.tsx (hunk 4 above)
Comment (bacebu4) [reply src/components/BaseInput.tsx:13]: I mean don't use `outline: none`

File: src/components/BaseInput.tsx (hunk 1 above)
Comment (bacebu4) [reply src/components/BaseInput.tsx:1]:
```suggestion
import { InputHTMLAttributes } from "react";
```

File: src/components/BaseInput.tsx (hunk 4 above)
Comment (bacebu4) [reply src/components/BaseInput.tsx:13]:
```suggestion
```

File: src/components/BaseInput.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import styled from "styled-components";
+import { GRAY, DARK } from "../utils/colors";
+
+const InputArea = styled.input<{ margin?: string }>`
+  margin: ${(props) => props.margin};
+
+  font-size: 1rem;
+  padding: 8px 12px;
+  background-color: ${GRAY}1A;
+  border-radius: 9px;
```
Comment (bacebu4) [inline src/components/BaseInput.tsx:11]:
```suggestion
  border-radius: 14px;
```

Comment (bacebu4) [reply src/components/BaseInput.tsx:11]: To match changed design

File: src/components/BaseInput.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import React, { InputHTMLAttributes } from "react";
+import styled from "styled-components";
+import { GRAY, DARK } from "../utils/colors";
+
+const InputArea = styled.input<{ margin?: string }>`
+  margin: ${(props) => props.margin};
+
+  font-size: 1rem;
```
Comment (bacebu4) [inline src/components/BaseInput.tsx:8]:
```suggestion
  font-size: 0.875rem;
```

**Comment by bacebu4** [comment]
ALSO add color variants according to new design

**Comment by bacebu4** [comment]
And don't forget to change margins
It's all what's left

**Comment by vlad99902** [comment]
I'll fix it. :)

**Review by bacebu4 — APPROVED** [review]

**Comment by bacebu4** [comment]
Went all wrong!

**Pull request closed without merging**